src/ddmd/mars.d

```diff
@@ -1682,11 +1691,10 @@ private bool parseCommandLine(const ref Strings arguments, const size_t argc, re
             {
                 if (p[8] == '=' && isdigit(cast(char)p[9]))
                 {
-                    errno = 0;
-                    const num = strtol(p + 9, &p, 10);
-                    if (*p || errno || num > INT_MAX)
+                    const num = parseDigits(p + 9, int.max);
+                    if (num == uint.max)
                         goto Lerror;
-                    global.errorLimit = cast(uint)num;
+                    params.errorLimit = num;
```

Comment:
This seems to assume that some other change has been merged.

Reply:
That's why there's the "blocked by" note above :-)